hcloud/util.go

```diff
@@ -57,7 +57,7 @@ func providerIDToServerID(providerID string) (id int, err error) {
 		return
 	}
 
-	idString := strings.Replace(providerID, providerPrefix, "", 1)
+	idString := strings.ReplaceAll(providerID, providerPrefix, "")
```

Comment:
Just to make sure: these two lines don’t do the same (first line replaces first occurrence, second line replaces all occurrences).

Reply:
This is intended.